packages/storybook/builder/src/types.ts

```diff
@@ -1,21 +1,18 @@
-import type { BuilderConfig as WebpackBuilderConfig } from '@modern-js/builder-webpack-provider';
-import type { BuilderConfig as RspackBuilderConfig } from '@modern-js/builder-rspack-provider';
-import { BuilderPlugin } from '@modern-js/builder-shared';
+import type { RsbuildPlugin } from '@rsbuild/shared';
```

Comment:
It is recommended to import types / methods from `@rsbuild/core`, as `@rsbuild/shared` is a private package of Rsbuild and may change in non-major versions.